src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/EncryptedXml.cs

```diff
@@ -185,7 +185,7 @@ private byte[] GetCipherValue(CipherData cipherData)
                 throw new ArgumentNullException(nameof(cipherData));
             }
 
-            Stream inputStream = null;
+            Stream? inputStream = null;
```

Comment:
I can't comment on it directly, but Recipient above also appears like it would benefit from an [AllowNull].  In general, if the setter allows a null to be set and the getter checks for null and returns non-null, the property can be non-nullable with [AllowNull].